test/test_static_file.rb

```diff
@@ -148,8 +148,9 @@ def setup_static_file_with_defaults(base, dir, name, defaults)
         "extname"       => ".txt",
         "modified_time" => @static_file.modified_time,
         "path"          => "/static_file.txt",
+        "collection"    => nil
```

Comment:
Can we add a test for a static file in a collection, please? That would help clarify this item's behavior.